languages/go/exercises/concept/constants/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+In this exercise you will establish some constants to be used in the operation of a bank.
```

Comment:
I can see how the first task fits in with the theme, but I feel thatthe other, date-related tasks make less sense. This might be due to the fact that the `constants` concept by itself is not _that_ interesting. Therefore, it might be convenient to combine this concept with another concept.

Reply:
Yeah this one was hard. Date-related tasks were used because dates are integral to the calculation of interest on loans and deposits. The `basics` concept is where variables are introduced and that would seem to be the natural destination for the `constants` concept if you don't want it to stand alone.

Reply:
@jamessouth That might be best discussed with the other Go maintainers. Looking at the amount of text in the introduction.md document, I don't think it makes sense to add it to the basics exercise as that would become too big I feel (unless only a basic constants introduction was given). An alternative would be to combine this concept with another concept.

Reply:
I think there is enough potential in constants to be a standalone exercise. Let us concentrate on merging this first, and then we can see how to improve it.

@jamessouth: could you address the open issues and then we can merge this?